tests/test_zarr_encryption_ipfs.py

```
@@ -7,26 +7,38 @@
 import pandas as pd
 import xarray as xr
 import pytest
+import zarr
+import json
 from numcodecs import register_codec
```

Ruff (0.8.2)

`numcodecs.register_codec` imported but unused

Remove unused import: `numcodecs.register_codec`

(F401)